src/charm.py

```diff
@@ -56,6 +68,31 @@ def apply_crds(self):
         self.unit.status = MaintenanceStatus("Applying CRDs")
         self.kubectl("apply", "-f", "templates/crd.yaml")
 
+    def apply_grafana_agent(self, remote_url):
+        patch_res = [
+            {"kind": "deployment", "name": "kube-ovn-monitor", "port": '"10661"'},
+            {"kind": "daemonset", "name": "kube-ovn-pinger", "port": '"8080"'},
+            {"kind": "daemonset", "name": "kube-ovn-cni", "port": '"10665"'},
```

Comment:
it's kinda interesting the port numbers are integers as strings as strings.  What's going on there?